src/main/java/com/google/sps/servlets/FindLocationServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/find-location")
+public class FindLocationServlet extends HttpServlet {
+
+ /**
+  * Given the correlator, this will update the database to reflect that the user clicked 'fiind nearby location'.
+  *
+  * @param  request  This includes a correlator for the current user.
```

Comment:
These param tags are fine to leave, but aren't actually usually needed for java unless you feel like you need to explain the variable more.

Theyre more useful in javascript, where you cant know the type of the inputs just by reading the code.

Reply:
I removed them